### DISCUSSION TITLE
[ Implemented ] Dimension monitoring without constraining element

### DISCUSSION THREAD

**Discussion by exegetor** [hlorus/CAD_Sketcher#234] [Feature Requests]
Per request on discord but also it's been on my own mind for a while...

I've had many occasions where I would like to see a particular dimension (distance or angle) but the sketch is already properly constrained.  Adding the dimension I want to monitor causes the solver to fail due to sketch being overconstrained.

I propose adding a option in the constraint list, allowing a constraint to act as a monitor only.  It wouldn't lock down an element but only report on its current dimension value.

**Comment by laundmo** [comment]
Im the onw who asked on discord, and heres my current usecase: the constraints are from real-world imperfect measurements which dont line up correctly. I would like to see what this angle is, to be able to check it against the real world source of truth - maybe the sides are in fact not paralell and this is just what i have to deal with.

![image](https://user-images.githubusercontent.com/24855949/181354267-a3189e35-5eae-4022-ab93-55bb75a11f61.png)

**Comment by hlorus** [comment]
Agree this would be a nice addition. And probably even trivial to implement...

**Comment by amrsoll** [comment]
See #272 :)

**Comment by TheMakerTales** [comment]
This has now been implemented.
It will be out in CAD SKetcher 0.26.0
All you'll have to do is click the measurement and set it to "Only Measure"
